nnue_bin_dataset.py

```diff
@@ -1,132 +0,0 @@
-import chess
```

Comment:
Can we leave this one around for environments we can't compile the c++ version?

Reply:
alternatively we could ship binaries...

Reply:
Know what, I'll bring back the old data loader tomorrow, make it sparse, just so it's there, because you have a point. I think it's a matter for another PR though.

Reply:
Sure, that's fine!  Let's get this rolling, it's a gigantic improvement :).